spec/models/post_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before :each do
+    @user = User.create(email: 'foobar@example.com', last_name: 'foobar', user_name: 'foobar', gender: 'male', first_name: 'foobar', password: 'foobar', password_confirmation: 'foobar')
```

Comment:
- [ ] **[OPTOINAL]** I suggest that you use [`factory_bot`](https://github.com/thoughtbot/factory_bot) in the coming tests to clean your specs.